clients/fides-js/src/lib/consent-utils.ts

```diff
@@ -226,16 +226,12 @@ const hasActionNeededTcfPreference = (
  */
 export const hasSavedTcfPreferences = (experience: PrivacyExperience) =>
   hasCurrentPreference(experience.tcf_purposes) ||
-  hasCurrentPreference(experience.tcf_special_purposes) ||
-  hasCurrentPreference(experience.tcf_features) ||
   hasCurrentPreference(experience.tcf_special_features) ||
   hasCurrentPreference(experience.tcf_vendors) ||
   hasCurrentPreference(experience.tcf_systems);
 
 export const hasActionNeededTcfPreferences = (experience: PrivacyExperience) =>
   hasActionNeededTcfPreference(experience.tcf_purposes) ||
-  hasActionNeededTcfPreference(experience.tcf_special_purposes) ||
```

Comment:
What about the state where an experience _only_ has `tcf_special_purposes` and `tcf_features`? We still want to show the banner in this case, and I believe removing these will prevent that from happening.

Reply:
oh hmm that's a good point! let me think about it...

Reply:
okay I think you are right, but this would require https://github.com/ethyca/fides/issues/4141. we basically want to treat them as notice-only, and instead of rendering "Save", "Opt out of all" "Opt into all" we just want the OK button, same as we have for notices. (it would be weird to surface something that only has special purposes and features and then have those buttons, since those aren't toggle-able!).

Reply:
hah yeah bit of a catch 22 but I'll go ahead and approve this, and we can follow-up with #4141, adding back these lines in that PR.